src/main/webapp/creatives/maps/simid_map_creative.test.js

```diff
@@ -38,8 +60,10 @@ beforeEach(() => {
         }
       };
     SimidProtocol.mockClear();
+    mockReject.mockReset();
     testMap = new SimidMapCreative();
     document.body.innerHTML = `
+    <div id="adContainer"></div>
```

Comment:
Is it okay that the button and map aren't nested inside the adContainer div here? Is there any difference in the tests when it's laid out this way?

Reply:
The tests don't alter the actual html. The only reason I needed to add this div is so that the tests recognized that this adContainer actually exists within the html